### PR TITLE
FileCacheUploadAPI

### PR DESCRIPTION
**Pull request by evans-sudo**

### CODE REVIEW

File: examples/main.go [hunk 1]
```diff
@@ -250,4 +255,86 @@ func SendBatchHTMLEmail() {
 	}
 
 }
+// file cache upload is
```
Comment (iqquee) [inline examples/main.go:258]: @evans-sudo Kindly complete the comment here

File: examples/main.go [hunk 2]
```diff
@@ -250,4 +255,86 @@ func SendBatchHTMLEmail() {
...
+	}
+
+	
+	for _, e := range res.Data {
+		fmt.Printf("response message: %v\n", e.Message)
+	}
+
+
+
+/**
```
Comment (iqquee) [inline examples/main.go:305]: @evans-sudo Kindly take the comments off since its not needed

File: mail.go [hunk 3]
```diff
@@ -132,3 +132,27 @@ func (c *Client) SendBatchHTMLEmail(req SendBatchHTMLEmailReq) (res *SendBatchHT
 	return &response, nil
 }
 
+func (c *Client) FileCacheUploadAPI(req FileCacheUploadAPIReq) (res *FileCacheUploadAPIRes, err error) {
+	url := "https://api.zeptomail.com/v1.1/files?name="
```
Comment (iqquee) [inline mail.go:136]: @evans-sudo Kindly update the URL to use the BaseURL from the client and properly fix the other parts of the URL

File: zeptomail.go [hunk 4]
```diff
@@ -57,6 +57,7 @@ func (c *Client) newRequest(method, reqURL string, reqBody, resp interface{}) er
 
 	req, err := http.NewRequest(method, newURL, body)
 	if reqBody != nil {
+	//	req.Header.Add("Content-Type", "text/plain")
```
Comment (iqquee) [inline zeptomail.go:60]: @evans-sudo Kindly take this off since its not needed

File: mail.go (hunk 3 above)
Comment (evans-sudo) [reply mail.go:136]: func (c *Client) FileCacheUploadAPI(req FileCacheUploadAPIReq) (res *FileCacheUploadAPIRes, err error) {
	url := c.BaseUrl
	Method := MethodPOST
	

is this okay?

**Pull request merged**